examples/examples_yaml_test.go

```diff
@@ -1384,299 +1280,6 @@ outputs:
 	}
 }
 
-// This replicates the diff when running `pulumi preview` on a aws.rds.Instance with
-// pulumi-aws v6.0.0 and state from pulumi-aws 5.42.0.
-//
-// The first test ensures we don't regress on https://github.com/pulumi/pulumi-aws/issues/2682
-//
-// The second test is when upgrading from pulumi-aws version <5.0.0 to v6.x.x, and
-// prevents regressions on https://github.com/pulumi/pulumi-aws/issues/2823
-//
-// Updated in https://github.com/pulumi/pulumi-aws/pull/3881
-// replacements.
-func TestMigrateRdsInstance(t *testing.T) {
```

Comment:
Removing this since it tested an issue with going from v5 to v6. Not relevant anymore for v6 to v7.